lib/common/utils.c

```diff
@@ -301,34 +301,43 @@ compare_version(const char *version1, const char *version2)
     return rc;
 }
 
+/*!
+ * \brief Parse milliseconds from a Pacemaker interval specification
+ *
+ * \param[in] input  Pacemaker time interval specification (a bare number of
+ *                   seconds, a number with a unit optionally with whitespace
+ *                   before and/or after the number, or an ISO 8601 duration)
+ *
+ * \return Milliseconds equivalent of given specification on success,
+ *         0 if input is NULL, or 0 (and set errno to EINVAL) on error
+ */
```

Comment:
It might make sense to note that the return value will be clamped to G_MAXUINT.

Reply:
Good point